src/components/payment/fee-summary.vue

```diff
@@ -52,16 +74,24 @@
             </v-slide-y-reverse-transition>
           </div>
         </div> -->
-
-        <div class="container fee-total payment-total pt-5" v-show="!fetchError">
-          <div class="fee-total__name">Total Amount (CAD)</div>
-          <!--<div class="fee-total__currency">CAD</div>-->
-          <div class="fee-total__value">
-            <v-slide-y-reverse-transition name="slide" mode="out-in">
-              <div><b>${{total.toFixed(2)}}</b></div>
-            </v-slide-y-reverse-transition>
-          </div>
-        </div>
+        <v-row no-gutters class="pt-4 pb-8">
+          <v-col cols="8">
+            <v-row no-gutters justify="start">
```

Comment:
Just curious, is this additional row required with only one element within?

Reply:
I thought it was required so I could use the justify "start", but maybe I could do this differently with the v-col. Will check

Reply:
There's probably a Verify helper class for that

Reply:
Alright so I can remove the v-rows for the ones I'm using for the ```justify="start"```, and for the other ones I will add ```style=float: right;``` to the div

Reply:
There's probably a float-right class already somewhere.

Reply:
I got in the habit of always using v-row for every item on the page because I thought it made adjusting things on the page easier to follow, but I guess it's not really needed and just adds more lines

Reply:
Your other option is to keep the v-rows so your code looks consistent, but add single v-cols under them.

Reply:
haha okay added the float-right class in. I'm good with this unless you would prefer the extra row/cols in there

Reply:
I'm good with this.

I'll leave it to Cam to resolve this convo.